content/en/tracing/trace_collection/custom_instrumentation/dotnet.md

```diff
@@ -25,22 +25,22 @@ further_reading:
       text: '.NET code samples'
 ---
 
+<div class="alert alert-info">
+If you have not yet read the instructions for auto-instrumentation and setup, start with the <a href="https://docs.datadoghq.com/tracing/setup/dotnet-core/">.NET/.NET Core</a> or <a href="https://docs.datadoghq.com/tracing/setup/dotnet-framework/">.NET Framework</a> Setup Instructions.
```

Comment:
Super nitpicky to change `auto-instrumentation` to `automatic instrumentation` as it is through the rest of the doc.

```suggestion
If you have not yet read the instructions for automatic instrumentation and setup, start with the <a href="https://docs.datadoghq.com/tracing/setup/dotnet-core/">.NET/.NET Core</a> or <a href="https://docs.datadoghq.com/tracing/setup/dotnet-framework/">.NET Framework</a> Setup Instructions.
```